### DISCUSSION TITLE
New Source: Google My Business

### DISCUSSION THREAD

**Discussion by nwayne94** [airbytehq/airbyte#33150] [New Connector Request]
## Tell us about the new connector you’d like to have
Google My Business --> S3 and/or Redshift

## Describe the context around this new connector
* Why do you need this integration? How does your team intend to use the data?
Our marketing team currently uses Google My Business to keep track of the various locations we service. This integration would allow us to pull My Business data into our data lake, similarly to what we are doing with our 10+ other data sources through Airbyte. We intend to utilize this data with data from other sources to create BI insights that help our marketing team formulate better campaigns, and to share high-level dashboards with our executive team
* How often do you want to run syncs?
~once every 24 hours
* If this is an API source connector, which entities/endpoints do you need supported?
https://developers.google.com/my-business/ref_overview
https://developers.google.com/my-business/reference/rest

## Describe the alternative you are considering or using
* What are you considering doing if you don’t have this integration through Airbyte?
Our marketing team will continue to use Google tools to locate the relevant insights, but it will remain separated from the rest of our data lake that already houses data from various other sources through Airbyte

## Are you willing to submit a PR?
Yes, but not until early next year, once some current projects are completed

**Comment by misteryeo** [comment]
**Issue was linked to Harvestr Discovery:**  [New Source: Google My Business](https://app.harvestr.io/components/0/list/EYtbdSQtv)

[a comment by an automated account is not shown]
